crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1793,17 +1808,13 @@ def admin_architecture
       #
       # openstackbackup helpers
       #
-      def postgres_params
+      def openstack_db_node
         db_node = ::Node.find("roles:database-config-default").first
```

Comment:
Maybe you can already check if there's a node with mysql-server role, otherwise we'd need to change this code later

Reply:
@jsuchome Why? The nodes with the "mysql-server" role will also have the "database-config-default" role. And making sure that the database-cluster is migrated to "mysql" should be part of the prechecks, shouldn't it?

Reply:
Ah, I missed that it's database-config-default and not database-server role you're looking for.